src/bp-blogs/bp-blogs-activity.php

```diff
@@ -352,7 +350,7 @@ function bp_blogs_format_activity_action_new_blog_comment( $action, $activity )
 		$comment = get_comment( $activity->secondary_item_id );
 		restore_current_blog();
 
-		if ( ! empty( $comment ) && ! is_wp_error( $comment ) ) {
+		if ( ! empty( $comment ) && is_a( $comment, 'WP_Comment' ) ) {
```

Comment:
I believe it's best to use `instanceof`.

```suggestion
		if ( ! empty( $comment ) && $comment instanceof WP_Comment ) {
```